tests/benchmark/compute/instruction/test_system.py

```diff
@@ -697,7 +695,11 @@ def test_selfdestruct_created(
 
     iterations = (gas_benchmark_value - base_costs) // loop_cost
 
-    code_prefix = Op.MSTORE(0, initcode.hex()) + Op.PUSH0 + Op.JUMPDEST
+    code_prefix = (
```

Comment:
TL;DR of bytecode changes is using CALLDATA 0 to resume in desired address target due to tx partitioning, and CALLDATA 1 for the desired count.

Reply:
What is the calldata encoding? We cannot resume at a desired address because we are using `Op.CREATE` (not `CREATE2`). 

I actually think for this test it is enough to just test for `Op.GT(Op.GAS, suffix_cost + loop_cost)`. We let the EVM decide if it has enough gas left to perform another loop (and pay enough for the SSTORE). So we do not have to encode any details into calldata.

(We can thus even use the "normal" tx splitter here without adding the calldata split logic)